### ISSUE TITLE
Bump PHP version to 7.1 to allow 'void' return type

### ISSUE THREAD

**Issue by holtkamp** [link0/bunq#28]
Might be an idea to bump the required version a bit, so PHPDoc can be cleaned up a bit 😄 

https://github.com/link0/bunq/blob/0305a271c03eccb0928ce26bb66f568edb24de71/composer.json#L17

**Comment by dennisdegreef** [comment]
Hi @holtkamp 

This seems like a good idea, although can you provide me some examples in which void/nullables are 'that' necessary to exclude 7.0.X PHP versions from being able to install?

We can then make an informed decision on the added benefit versus compatability.

Thanks!

**Comment by holtkamp** [comment]
>can you provide me some examples in which void/nullables would add real benefit

hehe, no, not really 😄 , but as soon you start to use them you will realize why, static analysis tools like PHPStan really start to add value. Stricter code allows less room for errors in my opinion / experience...

Also see https://laracasts.com/series/whats-new-in-php-7-1/episodes/3

**Comment by dennisdegreef** [comment]
I agree that stricter code reduces error, and I know what value they add :) I just wonder how much of an added value it provides in comparison with how much users (still on 7.0) you would rule out being able to use this library.

I think I will bump the requirement after a while, and keep a few versions compatible.
If you (or anyone for that matter) have suggestions for where this 'sweetspot' should be, feel free to comment.

**Comment by holtkamp** [comment]
ok, the difference between 7 and 7.1 is quite small IMHO while it does add a lot of nice features... While this project / library is young and no stable version is released, you can do whatever you want, right 😄
